### ISSUE TITLE
Could We Leverage `conventional-recommended-bump`?

### ISSUE THREAD

**Issue by hutson** [semantic-release/commit-analyzer-v2#11]
Today I came across [conventional-recommended-bump](https://www.npmjs.com/package/conventional-recommended-bump), and realized that it does most of what this `commit-analyzer` project does. Not only will it suggest a version bump, but it also supports presets. Each preset describes whether a commit type will cause a patch, minor, or major version bump.

I believe we could leverage the presets in `conventional-recommended-bump` to resolve discussions such as #3 

Each consumer of `semantic-release` could simply select the preset that corresponds with the commit types they want to release.

**Comment by jzsfkzm** [comment]
@boennemann what do you think about this idea? Either this change or making it configurable in commit-analyzer (#13) would be a great help in projects I'm working on currently.

**Comment by hutson** [comment]
Closing out this issue. Please feel free to re-open or re-file if switching to `conventional-recommended-bump` is still desired.

**Comment by stjohnjohnson** [comment]
@boennemann @christophwitzko any updates on this? 

@hbetts why did you close this?  What's an alternative?

**Comment by hutson** [comment]
@stjohnjohnson there is no immediate alternative if you're using `semantic-release`. You're only option is to wrap your preferred method using the plugin interface supported by `semantic-release`, and replacing the `analyzeCommits` plugin - https://github.com/semantic-release/semantic-release#analyzecommits

I closed out this issue because the lack of feedback caused me to pursue my own approach to automated releases. I originally came up with [semantic-release-github](https://www.npmjs.com/package/semantic-release-github), now deprecated, but now I use and maintain [semantic-release-gitlab](https://www.npmjs.com/package/semantic-release-gitlab).

Since there has been no feedback here, and I no longer have vested interest in this issue, I decided to close the issue out.
